api/datadoghq/v1alpha1/datadogpodautoscaler_types.go

```diff
@@ -191,6 +191,14 @@ type DatadogPodAutoscalerScalingPolicy struct {
 	// +listType=atomic
 	// +optional
 	Rules []DatadogPodAutoscalerScalingRule `json:"rules,omitempty"`
+
+	// StabilizationWindowSeconds is the number of seconds the controller should lookback at previous recommendations
+	// before deciding to apply a new one. Defaults to 0.
+	// +optional
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=1800
+	// +kubebuilder:default:0
```

Comment:
You don't need to set it, it's implicit.

Reply:
Good point, updated the minimum to 0 and removed the default setting!